cfg/cfg.go

```diff
@@ -104,6 +112,80 @@ type Filter struct {
 	Min uint32 `json:"min"`
 	Max uint32 `json:"max"`
 }
+type IPAPIResponse struct {
```

Comment:
This and the following lines are definitions which are IMO not related to config.
This is also true for the Aggrgate interface / struct definitions above, not sure if they came in later or are original ava code.
Please find a better place for defining them

Reply:
Ok, changed these structures from cfg.go to collections.go